Jenkinsfile

```diff
@@ -41,25 +41,49 @@ pipeline {
            }
            }
         }
-
         stage("fin get solver"){
-	      agent{  label "gfx908" }
+        agent{  label utils.rocmnode("tunatest") }
         steps {
             script {
             utils.finSolvers()
             }
             } 
         }
         stage("fin applicability"){
-	      agent{  label "gfx908" }
+        //init_session called here
```

Comment:
We can update init_session to take in arch/num_cu and run this stage on the no-gpu node as well